### ISSUE TITLE
Clarify / provide example on how to give correct credit.

### ISSUE THREAD

**Issue by Snapstromegon** [github/mona-sans#11]
Currently the website states that Mona Sans is "Licensed under [OFL](https://scripts.sil.org/OFL) (basically, use it on your site or in your app and credit GitHub—[read the full license](https://github.com/github/mona-sans/blob/main/LICENSE))". This is great, but the license is not that easy to understand, especially the case of a webpage distributing the font is handled under condition 2:

> 2) Original or Modified Versions of the Font Software may be bundled,
redistributed and/or sold with any software, provided that each copy
contains the above copyright notice and this license. These can be
included either as stand-alone text files, human-readable headers or
in the appropriate machine-readable metadata fields within text or
binary files as long as those fields can be easily viewed by the user.

It would be great to either on the website, or in the README (or both) have an example (or a more detailed description) of what kind of attributions are okay.

These could be like *To provide the required copyright notice, include this link on every page or in your legals page &lt;snippet of html&gt;*.

This would make it easier and faster for developers (like me) to include/use this font correctly without remaining doubts about legal issues and it would also make it more obvious that these kinds of links are required (especially when coming directly to the readme and not via the website).

The same obviously also applies to [github/hubot-sans](https://github.com/github/hubot-sans).

**Comment by justinpenner** [comment]
Attribution is not required under the OFL license, so I think if you want to give credit you can do it however you want. But perhaps someone at Github can suggest how to phrase the credit, if they have a preference.

If you're going to give credits, I think attributing the designer is important, too. I would write something like this:

> Typeface: [Mona Sans](https://github.com/mona-sans) by [Github](https://github.com/) & [Degarism Studio](https://degarism.com/).

As for the OFL's copyright notice and license requirements, they should already be embedded in the ttf/otf/woff/woff2 binaries, so that the requirement is fulfilled when serving fonts embedded in a web page. However, if you're distributing OFL-licensed fonts as a downloadable/shareable file, best practise is to include the license text file alongside the font binaries, bundled together in a .zip file.

**Comment by Snapstromegon** [comment]
According to the OFL license (at least as I read it in their FAQ), using them as web fonts is treated as distributing, not embedding.
Is there a (simple) tool out there to see the license data in a woff2 file? A quick Google search wasn't successful for me.
Also, if the license data in a font file is enough, the line about "credit GitHub" imo is kinda misleading.

**Comment by justinpenner** [comment]
> According to the OFL license (at least as I read it in their FAQ), using them as web fonts is treated as distributing, not embedding.

That's correct, I said "embedding", but "distributing" is the term OFL uses for web fonts.

> Is there a (simple) tool out there to see the license data in a woff2 file? A quick Google search wasn't successful for me.

[Font Drop](https://fontdrop.info/) will show you the metadata of a font, including the copyright and license data in the font's `name` table. It looks like Mona and Hubot have copyright info embedded, but they _do not_ have any embedded information about the license, so that's something that needs to be addressed.

Note that Font Drop doesn't support WOFF2 files for reasons not worth getting into here, but it's not hard to find a tool to decompress WOFF2 to OTF/TTF – I use [Google's command line tool](https://github.com/google/woff2).

> Also, if the license data in a font file is enough, the line about "credit GitHub" imo is kinda misleading.

I agree.

**Comment by Snapstromegon** [comment]
Thanks for your response and for sharing the links.
I also saw Font Drop, but I dismissed it because of the missing WOFF2 support, since I wanted to test the actual file I'd be shipping, so nothing gets lost in translation between formats (as I'm not an expert on font format internals).

I think some official input is required here going forward (and maybe we can get the license info embedded into the (WOFF2) files).
Maybe the codeowner @tobiasahlin could give chime in or ping the right people to help out.
